core/src/test/java/org/gbif/pipelines/core/ws/MockServer.java

```diff
@@ -1,4 +1,4 @@
-package org.gbif.pipelines.ws;
+package org.gbif.pipelines.core.ws;
```

Comment:
MockServer can be wrapped in a  @ClassRule and into a org.junit.rules.ExternalResource